example-get-started/code/src/evaluate.py

```diff
@@ -11,62 +11,75 @@
 from matplotlib import pyplot as plt
 
 
-live = Live("evaluation")
-
 if len(sys.argv) != 3:
     sys.stderr.write("Arguments error. Usage:\n")
     sys.stderr.write("\tpython evaluate.py model features\n")
     sys.exit(1)
 
 model_file = sys.argv[1]
-matrix_file = os.path.join(sys.argv[2], "test.pkl")
+train_file = os.path.join(sys.argv[2], "train.pkl")
+test_file = os.path.join(sys.argv[2], "test.pkl")
+
+def evaluate(model, matrix, dataset_name):
+    """Dump all evaluation metrics and plots for given datasets."""
```

Comment:
This is mostly the same as before, just put inside a function to reuse for train and test. There are two changes:
1. Paths were updated to be used across train and test.
2. Feature importance was moved elsewhere since it doesn't make sense to have train and test versions of it.